install.sh

```diff
@@ -32,6 +35,7 @@ gpgkey=http://packages.fluentbit.io/fluentbit.key
 enabled=1
 EOF
 yum install -y td-agent-bit wget
+yum install -y epel-release libmaxminddb-devel
```

Comment:
merge with 37  change to 
yum install -y epel-release td-agent-bit wget libmaxminddb-devel